.therapist.yml

```diff
@@ -7,6 +7,14 @@ actions:
       - .venv/
       - kinto/plugins/admin/node_modules/
 
+  isort:
+    run: isort --recursive --check-only .isort.cfg {files}
```

Comment:
I think you have to specify the virtualenv path to distinguish local from lib imports